eras/shelley-ma/test-suite/cardano-ledger-shelley-ma-test.cabal

```
@@ -1,6 +1,6 @@
 cabal-version: 3.0
 name:          cardano-ledger-shelley-ma-test
-version:       1.2.1.7
+version:       1.2.2.0
```

There ar eno changes to the package
```suggestion
version:       1.2.1.7
```